### DISCUSSION TITLE
Automatic upload of datakits to zenodo

### DISCUSSION THREAD

**Discussion by davide-f** [pypsa-meets-earth/pypsa-earth#633] [Ideas]
This methodology can be used to create a snakemake rule to upload data to zenodo: [link](https://towardsdatascience.com/zenodopy-manage-zenodo-projects-from-python-aa7c36634c1d)

**Comment by pz-max** [comment]
Looks promising! :) What about creating a feature request for this, explaining a bit more details of what needs to be done? (e.g. we have some notes in the dev meeting minutes)

**Reply by davide-f** [reply]
Issue added :)
I'll add more details later on
